keyboards/handwired/tractyl_manuform/5x6_right/rpi2040/post_config.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 Christopher Courtney, aka Drashna Jael're  (@drashna) <drashna@live.com>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#ifndef RGBLIGHT_LIMIT_VAL
+#    if defined(OLED_ENABLE)
+#        define RGBLIGHT_LIMIT_VAL 100
+#    else
+#        define RGBLIGHT_LIMIT_VAL 150
+#    endif
+#endif
+
+#ifndef OLED_BRIGHTNESS
+#    ifdef RGBLIGHT_ENABLE
```

Comment:
Since you have rgb matrix enabled here... 
```suggestion
#    if defined(RGBLIGHT_ENABLE) || defined(RGB_MATRIX_ENABLE)
```

Also, using amoebas or the like for the wiring?

Reply:
No, I'm using a flexible pcb. Only for the thumb cluster I use amoebas.